### DISCUSSION TITLE
Cleanvision for Video?

### DISCUSSION THREAD

**Discussion by ghost** [cleanlab/cleanvision#214] [Q&A] [answered]
Will there ever be a *video* version of `cleanvision` (i.e. do with video data what cleanvision can currently do with image data)? Or are there other tools that are better suited to working with *video* data?

**Comment by jwmueller** [comment, marked as answer]
Not on our current roadmap but would you like to help contribute an extension that make cleanvision runnable on video?
That sounds super useful to many folks! 
Should be fairly easy: 
- just extract every k-th frame from the video, 
- run cleanvision on those,
- aggregate the results across frames per video

**Reply by ghost** [reply]
Very interesting ... that might be something I could look into. Thanks again for clarifying!

**Reply by jwmueller** [reply]
awesome if you're ever interested in helping, I created a GH issue for this nice request:

https://github.com/cleanlab/cleanvision/issues/215

**Reply by ghost** [reply]
Excellent. I've already forked `cleanvision` and I'll be looking at the best way forward for *extending* the `imagelab` fuctionality to *video data*.
